pkg/receive/handler.go

```diff
@@ -79,7 +77,7 @@ func (h *Handler) Receive(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	req = req.WithContext(ctx)
-	req.Header.Add("THANOS-TENANT", r.Context().Value(authorize.TenantKey).(string))
```

Comment:
maybe we should make this configurable via a flag so we don't have to make a new PR to telemeter every time the tenant ID changes. WDYT?

Reply:
why would it change? it's kind of final ..

Reply:
when we run in different environments, or someone else runs this somewhere

Reply:
Hopefully, this is replaced by OIDC soo anyway, no? If so, we could just leave it as is and then update to the proper new flow later.